inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/sidebar/RecommendationSidebar.java

```diff
@@ -102,14 +107,29 @@ public RecommendationSidebar(String aId, IModel<AnnotatorState> aModel,
 
         add(new LearningCurveChartPanel(LEARNING_CURVE, aModel));
         
-        add(new RecommenderInfoPanel("recommenders", aModel));
+        recommenderInfos = new RecommenderInfoPanel("recommenders", aModel);
+        add(recommenderInfos);
     }
 
     @Override
     protected void onConfigure()
     {
         // using onConfigure as last state in lifecycle to configure visibility
         super.onConfigure();
+        configureMismatched();
+        AnnotatorState state = getModelObject();
+        if (state.getMode().equals(Mode.CURATION) 
```

Comment:
You could do 
`boolean enabled = state.getMode().equals(Mode.CURATION) || !state.getUser().equals(userRepository.getCurrentUser()))`
and then do

	     form.setEnabled(enabled);
        recommenderInfos.setEnabled(enabled);

Then it is not so convoluted to me. It would also be nice to have a comment here why we need that.

Reply:
Since the sidebar is only shown on the annotation page and not on the curation page, do we need to check the Mode at all?

Reply:
thx, good ideas, done